nonio-ios/Sources/APIs/Models/LoginResponse.swift

```diff
@@ -0,0 +1,8 @@
+
+
+import Foundation
+
+struct LoginResponse: Codable, Equatable {
+    let token: String
+    let username: String
```

Comment:
One thing I very recently added to the frontend was a `role` identifier to the login response (added this two days ago, which is why it was likely missed). The role identifier is an array of strings letting you know what capabilities the user has (ie admin, moderator, etc). Example response: 
![image](https://github.com/jjcm/nonio-ios/assets/50971/4305909f-2a1a-4eee-bc70-3805e91d89ab)

Reply:
Yes, the backend will return the `role` value, but currently, there is no implementation that utilizes this. Therefore, I have opted not to integrate it at this time. @jjcm